lib/discourse_code_review/github_repo.rb

```diff
@@ -162,10 +162,14 @@ def commits_since(ref = nil, merge_github_info: true, pull: true, single: false)
     end
 
     def followees(ref)
-      git_repo
-        .trailers(ref)
-        .select { |x| x.first == 'Follow-up-to' }
-        .map(&:second)
+      result = []
+
+      git_repo.commit(ref).message.lines.each do |line|
```

Comment:
Could you skip the first line, since this appears in revert messages?

Reply:
That's an interesting problem. I am not sure if I should always skip the first line or only do it when it starts with "REVERT:".

First solution will no longer recognize correctly messages like "Minor follow-up bug fix for 5ff6c10.", but the second one could theoretically be more problematic (I doubt it will be).

Reply:
Either way is fine with me, (but the prefix is `Revert "`)

Reply:
I think we can be even more lenient by checking for both space and colon `/^revert[: ]/i`